ci-operator/step-registry/baremetal/lab/agent/gather/baremetal-lab-agent-gather-commands.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+if test -f "${SHARED_DIR}/install-status.txt"; then
+  EXIT_CODE=$(cat "${SHARED_DIR}/install-status.txt")
+  if [ "$EXIT_CODE" == 0 ]; then
+    echo "Installation has been successfully completed, so there is no need to collect agent gather logs."
+    exit "${EXIT_CODE}"
+  fi
+fi
+
+if ! test -f "${SHARED_DIR}/node-zero-ip.txt"; then
+    echo "Installation method is different from the ABI, so there is no need to execute this step."
+    exit 0
+fi
+
+node_zero_ip=$(<"${SHARED_DIR}"/node-zero-ip.txt)
+SSH_PORT=2222
+SSHOPTS=(-o 'ConnectTimeout=5'
+  -o 'StrictHostKeyChecking=no'
+  -o 'UserKnownHostsFile=/dev/null'
+  -o LogLevel=ERROR
+  -i "${CLUSTER_PROFILE_DIR}/ssh-key")
+
+echo "Trying to gather agent logs on the host ${node_zero_ip}"
+ssh "${SSHOPTS[@]}" -N -L "${SSH_PORT}":"${node_zero_ip}":22 root@"${AUX_HOST}" &
```

Comment:
Hey @mhanss let's avoid doing tunneling if possible. See https://github.com/openshift/release/pull/47240/files#diff-e8f157108c4e736639dbbeeb84d5ac1b1f9da2291a126f7313f1eb8796891895R300-R303 for how to access nodes through the AUX_HOST and the already-configured port-forwarding.